lib/dialects/sqlite3/transaction.js

```diff
@@ -0,0 +1,12 @@
+const Transaction = require('../../execution/transaction');
+
+class Transaction_Sqlite extends Transaction {
+  begin(conn) {
+    const setIsolationLevel = this.isolationLevel
```

Comment:
Does it have to be different from how it's done in PG, which has no ternary operator and `Promise.resolve`?